trigger_class.py

```diff
@@ -0,0 +1,65 @@
+def contains(message, list_of_phrases):
+	"""
+	Checks if a message (string) contains all 
+	"""
+	for phrase in list_of_phrases:
+		if not contains_phrase(phrase):
+			return False
+
+	return True
+	
+
+def contains_phrase(message, phrase):
+	"""
+	Helper for checking if a phrase is in a message
+	"""
+	if phrase in message:

```

Comment:
This shouldn't work when a phrase is part of a word, but it does. Ex: phrase="weed" should not trigger on "weeder class"